src/commands/common/validate-asset-command.js

```diff
@@ -96,13 +101,82 @@ class ValidateAssetCommand extends Command {
             }
         }
 
+        let paranetId;
+        if (storeType === LOCAL_STORE_TYPES.TRIPLE_PARANET) {
+            try {
+                const {
+                    blockchain: paranetBlockchain,
+                    contract: paranetContract,
+                    tokenId: paranetTokenId,
+                } = this.ualService.resolveUAL(paranetUAL);
+
+                paranetId = this.paranetService.constructParanetId(
+                    paranetBlockchain,
+                    paranetContract,
+                    paranetTokenId,
+                );
+
+                if (!paranetId || paranetId === ZERO_BYTES32) {
```

Comment:
I don't think we need this, we should just check paranetUAL before doing anything

Reply:
Done